services/wallet/token/tokenstore.go

```diff
@@ -1679,6 +1679,14 @@ func newDefaultStore() *DefaultStore {
 				Decimals: 18,
 				ChainID:  11155111,
 			},
+			&Token{
```

Comment:
hmm is this the only "important" token available in that chain? not even DAI?

Reply:
I haven't found DAI!